WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.swift

```diff
@@ -89,4 +129,48 @@ private extension ReaderCommentsViewController {
         static let authorBadgeText = NSLocalizedString("Author", comment: "Title for a badge displayed beside the comment writer's name. "
                                                        + "Shown when the comment is written by the post author.")
     }
+
+    enum ReaderCommentMenu {
+        case unapprove
+        case spam
+        case trash
+        case edit
+        case share
+
+        var title: String {
+            switch self {
+            case .unapprove:
+                return NSLocalizedString("Unapprove", comment: "Unapproves a comment")
+            case .spam:
+                return NSLocalizedString("Spam", comment: "Marks comment as spam")
+            case .trash:
+                return NSLocalizedString("Trash", comment: "Trashes the comment")
```

Comment:
In Zeplin, these titles are:
- `Mark as Spam`
- `Mark as Trash`

Reply:
Thank you for noticing this! I don't know why I missed this 🤦 . This is fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17601/commits/4d621be16b989d3890b4581d540a62e4b094570e.

Reply:
Well it's a good thing you checked because I got it wrong too! 😆